core/src/main/java/org/zenoss/zep/impl/TriggerPlugin.java

```diff
@@ -848,7 +848,7 @@ protected synchronized void processSpool(long processCutoffTime) {
 
                 // These should have been deleted when the event was run through the TriggerPlugin when the status
                 // changed, but just in case delete them as the event is now in a closed state.
-                if (!OPEN_STATUSES.contains(status)) {
+                if (!OPEN_STATUSES.contains(status) || status == EventStatus.STATUS_ACKNOWLEDGED) {
```

Comment:
I was wondering if it where possible to specify whether the trigger fires based an event status would be better?  I wonder if there's another customer who'll be surprised that acknowledged events no longer fire event triggers.